server/src/pages/shot/view.js

```diff
@@ -465,7 +465,11 @@ class Body extends React.Component {
   }
 
   onClickOrigUrl(label) {
-    sendEvent("view-original", label, {useBeacon: true});
+    if (this.props.isOwner) {
+      sendEvent("view-original", label, {useBeacon: true});
+    } else {
+      sendEvent("goto-original-url", label, {useBeacon: true});
+    }
```

Comment:
Usually we use the label to specify owner or non-owner. Since we're passing in a label, let's just append it there, like `label + "-owner"`, and leave the action ("view-original") alone.

Also this should have a METRICS.md update